packages/ui-utils/src/shortenAddress.ts

```diff
@@ -0,0 +1,6 @@
+export const shortenAddress = (address: string, start = 10, end = 8) => {
+  if (!address) return '';
+  return address.length > start + end
+    ? `${address.slice(0, start)}...${address.slice(-end)}`
+    : address;
+};
```

Comment:
We have already such a [component](packages/epics/src/people/components/eth-address.tsx)